app/installer.sh

```diff
@@ -112,63 +109,12 @@ main() {
       err "Must have github's gh CLI installed to install a preview version."
     fi
 
-    inner() {
-      # shellcheck disable=SC2016
-      gh api graphql \
-        -f owner="$REPO_ORG" \
-        -f repo="$REPO_REPO" \
-        -f branch="$BRANCH" \
-        -F "count=${1}" \
-        -q '.data.repository.ref.target.history.nodes | map(
-            select(
-              .statusCheckRollup.state == "SUCCESS"
-            )
-          ) | first | .oid
-        ' \
-        -F query='
-          query(
-            $branch: String!,
-            $owner: String!,
-            $repo: String!,
-            $count: Int!
-          ) {
-            repository(
-              name: $repo,
-              owner: $owner
-            ) {
-              ref(qualifiedName: $branch) {
-                target {
-                  ... on Commit {
-                    history(first: $count) {
-                      nodes {
-                        oid
-                        statusCheckRollup {
-                          state
-                        }
-                      }
-                    }
-                  }
-                }
-              }
-            }
-          }
-        ' ||
-        true
-    }
-
-    for count in 5 10 25 50; do
-      commit="$(inner "$count")"
-
-      if echo "$commit" | grep -qs -E '[a-z0-9]{9,}'; then
-        break
-      fi
-    done
-
+    log "Fetching latest preview commit.."
+    commit=$(gh run list --repo "${REPO}" --branch "${BRANCH}" --workflow build-test-distribute -L 200 --json event,headSha,conclusion --jq '[.[] | select(.conclusion == "success" and .event == "push")] | first | .headSha[0:9]')
```

Comment:
Seems like `--branch` is not available: https://github.com/kumahq/kuma-website/actions/runs/8570497791/job/23488536534?pr=1710#step:6:15

Reply:
Ha nice! Thx for digging in!

Reply:
Opened: https://github.com/kumahq/kuma-website/issues/1711 to track this